lib/proto/xudrpc.swagger.json

```diff
@@ -21,7 +21,7 @@
         "operationId": "AddCurrency",
         "responses": {
           "200": {
-            "description": "",
+            "description": "A successful response.",
```

Comment:
Can you try regenerating protos to see if it gets rid of some of these unintended changes?